app/src/main/java/com/github/se/studybuddies/ui/topics/TopicResources.kt

```diff
@@ -173,3 +230,68 @@ private fun UserBox(user: User) {
         }
   }
 }
+
+@Composable
+fun AddResources(
+    topicFileViewModel: TopicFileViewModel,
+    showOptions: MutableState<Boolean>,
+    showUploadImage: MutableState<Boolean>,
+    showUploadLink: MutableState<Boolean>,
+    showUploadFile: MutableState<Boolean>,
+) {
+  PickPicture(showUploadImage) { topicFileViewModel.addImage(it.value) }
+  // UploadLink(viewModel, showUploadLink)
+  // UploadFile(viewModel, showUploadFile)
+  ShowAlertDialog(
+      showDialog = showOptions,
+      onDismiss = { showOptions.value = false },
+      title = {},
+      content = {
+        Row(
```

Comment:
this dialog seems quite big with the icon very small in the center, but i guess it is related to the commented code that is not displayed here yet

Reply:
yeah exactly. I'm leaving room for future development but I agree that it lookds a bit big now